test/e2e/kubernetes/scripts/cloud-init-files-validate.sh

```diff
@@ -15,4 +16,6 @@ done
 # verify that no files under /var/log have read access to everyone
 sudo find /var/log -type f -perm '/o+r' | (! grep ^) || exit 1
 # specific file mode validations
-stat /boot/grub/grub.cfg | grep 'Access: (0400' || exit 1
\ No newline at end of file
+stat /boot/grub/grub.cfg | grep 'Access: (0400' || exit 1
+# validate grub configuration
+sudo grep "^\s*linux" /boot/grub/grub.cfg | grep 'audit=1' || exit 1
```

Comment:
Should we also grep for `ipv6.disable=1` or is that not part of the CIS recommendation?

Reply:
Yeah, in fact this grep is carried over specifically from here:

https://secscan.acron.pl/centos7/4/1/3